user-management/user-details/script.js

```diff
@@ -625,6 +629,218 @@ function generateUserIdleDetails() {
   document.querySelector('.accordion-availability').append(div);
 }
 
+function formatDate(dateStr) {
+  const date = new Date(dateStr);
+  const options = {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+    hour: '2-digit',
+    minute: '2-digit',
+    timeZone: 'Asia/Kolkata',
+  };
+  const formattedDate = date.toLocaleDateString('en-US', options);
+  return `${formattedDate} IST`;
+}
+
+function generatePRsTabDetails() {
+  const div = createElement({
+    type: 'div',
+    classList: ['hidden-content', 'hide'],
+  });
+  const prsSec = createElement({
+    type: 'div',
+    classList: ['user-pr'],
+  });
+  const pagination = createElement({ type: 'div', classList: ['pagination'] });
+  const prevBtn = createElement({
+    type: 'button',
+    classList: ['pagination-load-more'],
+  });
+  prevBtn.appendChild(createTextNode('Load More'));
+  prevBtn.addEventListener('click', loadMorePRs);
+  pagination.append(prevBtn);
+  div.append(prsSec, pagination);
+  document.querySelector('.accordion-prs').appendChild(div);
+}
+
+async function getUserPRs() {
+  try {
+    const res = await makeApiCall(
+      `${API_BASE_URL}/pullrequests/user/${username}`,
```

Comment:
This URL can go inside constants

Reply:
Is it good to create constants for every url because `API_BASE_URL `is already **constants** ?